source/exploring_catalog_and_datasets/04_getting_involved/getting_involved.rst

```diff
@@ -0,0 +1,29 @@
+Getting involved: sharing, reusing & reacting
+=============================================
+
+As a data consumer, multiple options are available to get more involved into the open data environment. Whether you come accross some datasets that you are particularly interested in, and want to share them with the world. Or whether you have specific skills you believe could be put to use to create something with those published data.
```

Comment:
> As a data consumer, multiple options are available to get more involved into the open data environment. 

As a data consumer, multiple options are available to get more involved **in** the open data environment.

Reply:
> Whether you come accross some datasets that you are particularly interested in, and want to share them with the world. Or whether you have specific skills you believe could be put to use to create something with those published data.

the open data _environment: whether_ you come accross some datasets that you are particularly _interested in and want to share_ with the _world, or_ whether you have specific skills you believe could be put to use to create something with _the_ published data.